plenum/common/messages/fields.py

```diff
@@ -181,6 +188,14 @@ def _specific_validation(self, val):
                 return val_error
 
 
+class HetroMapField(FieldBase):
```

Comment:
For me 'Hetro' doesn't look clear. Probably a full name (Heterogeneous) can be used.
Also the code just shows that a dict can be any type (no specific validation), so for me AnyMap looks fine.